cmd/atlas/internal/cmdext/cmdext.go

```diff
@@ -152,6 +156,24 @@ func AWSRDSToken(ctx *hcl.EvalContext, block *hclsyntax.Block) (cty.Value, error
 	if err != nil {
 		return cty.NilVal, errorf("building auth token: %v", err)
 	}
+	// Database network traffic is encrypted using SSL/TLS when using IAM on mysql RDS.
+	// So we need to download the RDS CA certificate, and register it with the mysql driver.
+	resp, err := http.DefaultClient.Get("https://s3.amazonaws.com/rds-downloads/rds-combined-ca-bundle.pem")
+	if err != nil {
+		return cty.NilVal, errorf("downloading ca bundle: %v", err)
+	}
+	pem, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return cty.NilVal, errorf("reading ca bundle: %v", err)
+	}
+	rootCertPool := x509.NewCertPool()
+	if ok := rootCertPool.AppendCertsFromPEM(pem); !ok {
+		return cty.NilVal, errorf("appending certs from pem: %v", err)
+	}
+	err = mysql.RegisterTLSConfig("rds", &tls.Config{RootCAs: rootCertPool, InsecureSkipVerify: true})
```

Comment:
I don't like working with drivers directly in packages that are not main (atlas packages don't aware to the underlying driver). I prefer creating a driver for `awsmysql`. e.g., https://github.com/google/go-cloud/blob/master/mysql/awsmysql/awsmysql.go

Reply:
Is it necessary? I'm guessing its posible to provide the CA path to the driver. 
we shouldn't do the magic of forcing the user to trust some cert. its an advanced use case and the user should manage their own CAs.

Reply:
> Is it necessary? I'm guessing its posible to provide the CA path to the driver. we shouldn't do the magic of forcing the user to trust some cert. its an advanced use case and the user should manage their own CAs.

Yes. It is needed. This is how it works with mysql go sql driver sadly.